nullaway/src/main/java/com/uber/nullaway/AbstractConfig.java

```diff
@@ -338,11 +345,6 @@ public String getErrorURL() {
     return errorURL;
   }
 
-  @Override
-  public boolean treatGeneratedAsUnannotated() {
-    return treatGeneratedAsUnannotated;
-  }
```

Comment:
We basically ended up duplicating this, because the original name of the method didn't match the name of the field/option 😅